ideas-conference-planner/main.js

```diff
@@ -363,6 +363,10 @@ ipc.on('query-presentations', function(event, arg) {
   event.returnValue = queryPresentations(event);
 });
 
+ipc.on('get-category', function(event, arg) {
```

Comment:
Is this a mistake? Having a blank function and then the exact same `ipc.on` call right below with information?

Reply:
the value is set when the dropdown is changed.  You're right, i took it out.

Reply:
the first category is supposed to be an empty space, it isn't possible to access it. 
We never got to the editing of the categories.

Reply:
If the first category is supposed to be an empty space, then it doesn't work for me. I have a single category in the database and cannot set any categories to anything. I don't have an empty space in the drop down. It's just the single category. And nothing is every set.